custodian/vasp/tests/conftest.py

```diff
@@ -12,3 +12,13 @@ def _patch_get_potential_energy(monkeypatch):
     Monkeypatch the multiprocessing.cpu_count() function to always return 64
     """
     monkeypatch.setattr(multiprocessing, "cpu_count", lambda *args, **kwargs: 64)
+
+
+@pytest.fixture(autouse=True)
+def _clear_tracked_cache():
+    """
+    Clear the cache of the stored functions between the tests.
+    """
+    from custodian.utils import tracked_lru_cache
+
+    tracked_lru_cache.tracked_cache_clear()
```

Comment:
I think this fixture currently runs on _every_ test, not just the handler tests which would be unnecessary work?

Reply:
I think you can move it to `custodian/vasp/handlers.py` but keep `autouse=True` [to only auto apply it to those tests](https://docs.pytest.org/en/6.2.x/fixture.html#fixture-availability).

Reply:
Since this is also used in the validators, I think the fixture should either be duplicated in the handlers and validators files, or added it explicitly to the tests that need it, removing the `autouse`.

Reply:
Yeah, probably duplicating it in both files makes it more explicit too. People are more likely to see it and become aware this is happening.